<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test3Class</class>
 <widget class="QMainWindow" name="test3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="itemsListWidget"/>
      </item>
      <item>
       <widget class="QListWidget" name="filteredListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="categoryLabel">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="categoryLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="filterPushButton">
          <property name="text">
           <string>Show Items</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchPushButton</sender>
   <signal>clicked()</signal>
   <receiver>test3Class</receiver>
   <slot>searchItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterPushButton</sender>
   <signal>clicked()</signal>
   <receiver>test3Class</receiver>
   <slot>filterItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchItem()</slot>
  <slot>filterItems()</slot>
 </slots>
</ui>
